docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -392,85 +497,62 @@ Accept: application/json
-| Check Connection Options URL    | A URL for a shipping carrier resource that accepts check requests containing the connection options provided by a user when enabling the carrier and indicates whether or not those settings are valid. This is an optional step.                                                                                                                                                                                                                                                                                               |
-| Shipping Quote                  | An estimation of the cost to ship a set of items from an origin to a destination.                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| Shipping Zone                   | Describes a set of destination addresses and the applicable shipping settings, such as handling fees and available shipping methods.                                                                                                                                                                                                                                                                                                                                                                                            |
-| Shipping Origin                 | The location from which goods are shipped. This origin determines which shipping carriers are available for the merchant to configure in the control panel.                                                                                                                                                                                                                                                                                                                                                                     |
+| Name | Description |
+| ---- | ---- |
+| Configuration Fields | Optional connection and shipping settings fields. Merchants and API users use these fields to connect your carrier to their store and define define shipping methods for your carrier in a zone. |

```

Comment:
Does it make sense to define `connection options` and `settings options` (or whatever nomenclature makes sense) separately, then just define `configuration fields` as a generic term to describe both?

Or am I just making it more confusing?